web/src/user/LibraryPage/ApplicationEmptyState.ts

```diff
@@ -19,7 +20,7 @@ import PFSpacing from "@patternfly/patternfly/utilities/Spacing/spacing.css";
  * administrator, provide a link to the "Create a new application" page.
  */
 
-const styles = [
+const styles = adaptCSS([
```

Comment:
Is this the only place in all of the components where the wrapper needs to be added?